### PR TITLE
Support LLVM 17

### PR DESCRIPTION
**Pull request by SUPERCILEX**
Taken from and closes https://github.com/TheDan64/inkwell/pull/448, fixes https://github.com/TheDan64/inkwell/issues/444.

### CODE REVIEW

**Comment by SUPERCILEX** [comment]
@TheDan64 got it working!

File: src/values/mod.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ pub use crate::values::traits::AsValueRef;
 pub use crate::values::traits::{AggregateValue, AnyValue, BasicValue, FloatMathValue, IntMathValue, PointerMathValue};
 pub use crate::values::vec_value::VectorValue;
 
-#[llvm_versions(12.0..=latest)]
+#[llvm_versions(12.0..=16.0)]
 use llvm_sys::core::LLVMIsPoison;
```
Comment (TheDan64) [inline src/values/mod.rs:51]: Is poison no longer supported in LLVM17?

Comment (SUPERCILEX) [reply src/values/mod.rs:51]: Looks like it still is, do I'll revert that later tonight: https://docs.rs/llvm-sys/latest/llvm_sys/?search=LLVMIsPoison

Comment (SUPERCILEX) [reply src/values/mod.rs]: Actually that import wasn't even being used so I got rid of it.

**Review by TheDan64 — APPROVED** [review]

**Comment by TheDan64** [comment]
Thanks!

**Pull request merged**